spt/OrangeBox/modules/ServerDLL.cpp

```diff
@@ -440,7 +473,7 @@ bool __fastcall ServerDLL::HOOKED_CheckJumpButton_Func(void* thisptr, int edx)
 
 void __fastcall ServerDLL::HOOKED_FinishGravity_Func(void* thisptr, int edx)
 {
-	if (insideCheckJumpButton && y_spt_additional_abh.GetBool())
+	if (insideCheckJumpButton && y_spt_additional_jumpboost.GetInt())
```

Comment:
Why did you change this to `GetInt()`? It's identical to `GetBool()` here, and `GetBool()` is better.